CHANGELOG

```diff
@@ -63,6 +63,10 @@ Version 4.2.1 (development)
 - Added a new miniapp for computing (signed) distance functions to a point
   source or zero level set. See miniapps/shifted/distance.cpp.
 
+- Added support for some examples (ex1p, ex2p, ex7p) to use hypre built with
+  CUDA and unified memory, when MFEM is built with `make pcuda` and run with
+  `-d cuda:uvm` as a command-line option.
```

Comment:
Is the uvm necessary or could it also work with `cuda`?

Reply:
This bullet is out of date. Both with and without uvm should work.

Reply:
Also, more examples and miniapps are supported now, right?

Reply:
Yes, all examples that have the `-d/--device` option should work with `-d cuda` and `-d cuda:uvm`. Let me know if you notice any problems.

Reply:
I think this should be completely rewritten. How about the following

```
- Added support for AMG preconditioners on GPUs based on the hypre library
  (version 2.22.0 or later). These include BoomerAMG, AMS and ADS and most
  MFEM examples that use hypre have been ported to support this functionality.
  The GPU preconditioners require that both hypre and MFEM are built with CUDA
  support. Hypre builds with CUDA and unified memory are also supported and
  can be used with `-d cuda:uvm` as a command-line option.

- Extended `make test` to include GPU tests when MFEM is built with CUDA or HIP
  support.
```

```

Reply:
Sounds good to me.